### PR TITLE
Scholarly works worktype

### PR DESCRIPTION
**Pull request by smithjp**

### CODE REVIEW

File: config/locales/scholarly_work.en.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+en:
+  hyrax:
+    icons:
+      scholarly_work:     'fa fa-file-text-o'
+    select_type:
+      scholarly_work:
+        name:               "Scholarly Work"
+        description:        "Scholarly work works"
```
Comment (bbpennel) [inline config/locales/scholarly_work.en.yml]: As much as I enjoy "Scholarly work works", this and all the translations should probably have the extra "work" trimmed from them.

File: spec/controllers/hyrax/scholarly_works_controller_spec.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Generated via
+#  `rails generate hyrax:work ScholarlyWork`
+require 'rails_helper'
+
+RSpec.describe Hyrax::ScholarlyWorksController do
```
Comment (bbpennel) [inline spec/controllers/hyrax/scholarly_works_controller_spec.rb]: Looks like this can be deleted

File: spec/models/scholarly_work_spec.rb [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Generated via
+#  `rails generate hyrax:work ScholarlyWork`
+require 'rails_helper'
+
+RSpec.describe ScholarlyWork do
+  it "has tests" do
```
Comment (bbpennel) [inline spec/models/scholarly_work_spec.rb]: Should this be filled out similar to the spec for https://github.com/UNC-Libraries/hy-c/blob/master/spec/models/honors_thesis_spec.rb ?

File: config/locales/scholarly_work.en.yml (hunk 1 above)
Comment (smithjp) [reply config/locales/scholarly_work.en.yml]: I agree.  I will ask Julie and Rebekah what the visible names and descriptions should be.

**Pull request merged**